### PR TITLE
Adds energy bolas to the secvend.

### PR DESCRIPTION
**Pull request by TechnoAlchemisto**
How's that for scifi, am I right gamers?
:cl: Papa Bones
tweak: Security now has energy bolas available to them in their vending machines, check them out!
/:cl:
COCKANDBALLTORTURE

### CODE REVIEW

**Comment by optimumtact** [comment]
This is more a tweak since the item already existed

**Comment by TechnoAlchemisto** [comment]
I'll take it

File: code/game/objects/items/weapons/storage/belt.dm [hunk 1]
```diff
@@ -129,7 +129,8 @@
 		/obj/item/device/flashlight/seclite,
 		/obj/item/weapon/melee/classic_baton/telescopic,
 		/obj/item/device/radio,
-		/obj/item/clothing/gloves/
+		/obj/item/clothing/gloves/,
+		/obj/item/weapon/restraints/legcuffs/bola
```
Comment (PKPenguin321) [inline code/game/objects/items/weapons/storage/belt.dm]: stealth buff

File: code/game/objects/items/weapons/handcuffs.dm [hunk 2]
```diff
@@ -338,3 +338,20 @@
 	breakouttime = 70
 	origin_tech = "engineering=4;combat=3"
 	weaken = 1
+
+/obj/item/weapon/restraints/legcuffs/bola/energy //For Security
+	name = "Energy Bola"
+	desc = "A specialized hard-light bola designed to ensnare criminals and aide in arrests."
+	icon_state = "ebola"
+	weaken = 0
```
Comment (PKPenguin321) [inline code/game/objects/items/weapons/handcuffs.dm]: you dont need this line

File: code/game/objects/items/weapons/handcuffs.dm [hunk 3]
```diff
@@ -338,3 +338,20 @@
 	breakouttime = 70
 	origin_tech = "engineering=4;combat=3"
 	weaken = 1
+
+/obj/item/weapon/restraints/legcuffs/bola/energy //For Security
+	name = "Energy Bola"
+	desc = "A specialized hard-light bola designed to ensnare criminals and aide in arrests."
+	icon_state = "ebola"
```
Comment (PKPenguin321) [inline code/game/objects/items/weapons/handcuffs.dm]: meme

File: code/game/objects/items/weapons/handcuffs.dm [hunk 4]
```diff
@@ -338,3 +338,19 @@
...
+	hitsound = 'sound/weapons/taserhit.ogg'
+	w_class = 2
+	breakouttime = 60
+
+/obj/item/weapon/restraints/legcuffs/bola/energy/throw_impact(atom/hit_atom)
+	if(iscarbon(hit_atom))
+		var/obj/item/weapon/restraints/legcuffs/beartrap/B = new /obj/item/weapon/restraints/legcuffs/beartrap/energy/cyborg(get_turf(hit_atom))
+		B.Crossed(hit_atom)
+		qdel(src)
+	spawn(10)
```
Comment (AnturK) [inline code/game/objects/items/weapons/handcuffs.dm]: What's with this spawn

Comment (TechnoAlchemisto) [inline code/game/objects/items/weapons/handcuffs.dm]: I have no idea, I converted this from the peacekeeper energy bola and it was there and it worked so I assumed it was necessary.

**Pull request merged**